presto-orc/src/test/java/com/facebook/presto/orc/TestingColumnHandle.java

```diff
@@ -0,0 +1,68 @@
+/*
```

Comment:
I tried that but my editor refused to import it. Is that possible or am I doing something wrong?